firmware/targets/furi_hal_include/furi_hal_rtc.h

```diff
@@ -31,7 +31,8 @@ typedef enum {
     FuriHalRtcFlagC2Update = (1 << 3),
     FuriHalRtcFlagHandOrient = (1 << 4),
     FuriHalRtcFlagLegacySleep = (1 << 5),
-    FuriHalRtcFlagStealthMode = (1 << 6),
+    FuriHalRtcFlagFilenameMode = (1 << 6),
```

Comment:
Nope, you can not replace already existing flag

Reply:
Also mode in FuriHalRtcFlagFilenameMode kinda expects that there will be some options. Not sure how it should be called.

Reply:
Leaving this open, as I am unsure if this change is visual / the right fix. I made sure to keep FuriHalRtcFlagStealthMode, but added FuriHalRtcFlagFilenameScheme in the most recent commit.